source/Irrlicht/COpenGLExtensionHandler.h

```diff
@@ -2472,6 +2475,12 @@ inline void COpenGLExtensionHandler::setPixelUnpackAlignment(const uint32_t &pit
     pixelUnpackAlignment = textureUploadAlignment;
 }
 
+inline void COpenGLExtensionHandler::extGlCreateSamplers(GLsizei n, GLuint* samplers)
+{
+    if (pGlCreateSamplers)
+        pGlCreateSamplers(n, samplers);
```

Comment:
if you don't have DSA use generate instead

Reply:
resolved